libraries/Adapters/Microsoft.Bot.Builder.Adapters.Twilio/TwilioClientWrapper.cs

```diff
@@ -1,6 +1,7 @@
 ﻿// Copyright (c) Microsoft Corporation.All rights reserved.
```

Comment:
There's a space missing.
```suggestion
// Copyright (c) Microsoft Corporation. All rights reserved.
```